src/azure-cli/azure/cli/command_modules/vm/aaz/latest/disk/__cmd_group.py

```diff
@@ -18,7 +18,7 @@ class __CMDGroup(AAZCommandGroup):
     """Manage Azure Managed Disks.
 
     Azure Virtual Machines use disks as a place to store an operating system, applications, and data. All Azure virtual machines have at least two disks: An operating system disk, and a temporary disk. The operating system disk is created from an image, and both the operating system disk and the image are actually virtual hard disks (VHDs) stored in an Azure storage account. Virtual machines also can have one or more data disks, that are also stored as VHDs.
-    Azure Unmanaged Data Disks have a maximum size of 4095 GB. To use disks larger than 4095 GB use [Azure Managed Disks](https://learn.microsoft.com/azure/virtual-machines/managed-disks-overview)
+    Azure Unmanaged Data Disks have a maximum size of 4095 GB. To use disks larger than 4095 GB use [Azure Managed Disks](https://docs.microsoft.com/azure/virtual-machines/managed-disks-overview)
```

Comment:
Why do we need to modify the URL here?

Reply:
You are right. We don't need to change the URL.
I have no idea how it was changed. I'll change it back.